src/components/Avatar/Avatar.tsx

```diff
@@ -58,18 +59,18 @@ const sizeToFallbackTypeScale: Record<AvatarSize, Typography> = {
   xl: "xl3",
 };
 
-const sizeToIconInc = {
+const sizeToIconInc: Record<AvatarSize, number> = {
   sm: 2.5,
   md: 4,
   lg: 5,
   xl: 8,
 };
 
 const sizeToTypeScale: Record<AvatarSize, Typography> = {
-  sm: "baseMd",
-  md: "baseMd",
-  lg: "lg",
-  xl: "xl3",
+  sm: "smMd",
+  md: "smMd",
+  lg: "base",
+  xl: "base",
```

Comment:
Updated to match Figma: https://www.figma.com/file/aWUE4pPeUTgrYZ4vaTYZQU/%E2%9C%A8Beam-Component-Library?node-id=31578-99812&t=UDyUmfKZ9W6rXTdJ-0